File: src/main/java/org/example/Java/Act11.java
package org.example.Java;

import java.util.Scanner;

public class Act11 {
    Scanner scanner = new Scanner(System.in);
    public void Fibonachi() {
        System.out.println("Dime  los primeros N números de la secuencia de Fibonacci ?");
        int numbre = scanner.nextInt();
        int fibonacci = 0;
        int num1 = 0;
        int num2 = 1;

        for(var i=1;i<numbre+1;++i)
        {
            fibonacci=num1+num2;
            System.out.print(" "+fibonacci);
            num1=num2;
            num2=fibonacci;
        }

    }
}

File: src/main/java/org/example/Kotlin/Act11.kt
package org.example.Kotlin

class Act11 {
    fun Fibonachi() {
        println("Dime  los primeros N números de la secuencia de Fibonacci ?")
        val numbre: Int = readLine()!!.toInt()
        var fibonacci = 0
        var num1 = 0
        var num2 = 1

        for (i in 1 until numbre + 1) {
            fibonacci = num1 + num2
            print(" $fibonacci")
            num1 = num2
            num2 = fibonacci
        }
    }
}